package.json

```diff
@@ -37,9 +37,10 @@
   "scripts": {
     "predeploy": "yarn build",
     "deploy": "gh-pages -d dist/",
-    "start": "parcel src/index.html",
+    "update:static": "cp src/wasm/multisig.wasm dist/multisig.wasm",
```

Comment:
Why is this extra copy step needed? Why not just commit `dist/multisig.wasm` directly?

Reply:
`"prebuild": "rm -rf dist/"`
I usually don't trust a file will remain in `/dist` and cp static assets over on run/builds?